sdk/test-utils/recorder-new/CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## 1.0.0 (Unreleased)
 
```

Comment:
Looks like the builds are failing because testing-recorder-new needs corresponding updates.

Reply:
Ha, I realized that I forgot to update this when I was about to go to sleep last night. I'll update it in a sec